### PR TITLE
Updated kafka's zookeeper dependency

### PR DESCRIPTION
**Pull request by dkapanidis**
Also fixed some incorrect plumbing of the zookeeper service name

### CODE REVIEW

**Comment by lachie83** [comment]
@k8s-bot ok to test

File: incubator/kafka/templates/kafka-petset.yaml [hunk 1]
```diff
@@ -9,7 +9,7 @@ spec:
   - port: 2181
```
Comment (lachie83) [inline incubator/kafka/templates/kafka-petset.yaml:8]: Can you rename this file?

**Comment by lachie83** [comment]
Just waiting on the file to be renamed.

**Comment by dkapanidis** [comment]
The suffix `ss` is adopted as seen in zookeeper chart. Also other references of PetSet have been replaced with StatefulSet.

**Comment by lachie83** [comment]
LGTM

**Comment by viglesiasce** [comment]
@k8s-bot ok to test

**Pull request merged**